WordPress/Classes/ViewRelated/NUX/DomainSuggestionsTableViewController.swift

```diff
@@ -3,17 +3,35 @@ import SVProgressHUD
 import WordPressAuthenticator
 
 
-protocol SiteCreationDomainsTableViewControllerDelegate {
+protocol DomainSuggestionsTableViewControllerDelegate {
     func domainSelected(_ domain: String)
     func newSearchStarted()
 }
 
-class SiteCreationDomainsTableViewController: NUXTableViewController {
+/// This is intended to be an abstract base class that provides domain
+/// suggestions for the keyword that user searches.
+/// Subclasses should override the open variables to make customizations.
+class DomainSuggestionsTableViewController: NUXTableViewController {
 
     // MARK: - Properties
 
     open var siteName: String?
-    open var delegate: SiteCreationDomainsTableViewControllerDelegate?
+    open var delegate: DomainSuggestionsTableViewControllerDelegate?
+    open var suggestOnlyWordPressDotCom: Bool {
+        return true
+    }
+    open var useFadedColorForParentDomains: Bool {
```

Comment:
`useFadedColorForParentDomains` and `parentDomainColor` are not clear as to what they do. Also, they seem a bit unnecessary. It it possible to just have an `open var domainTextColor` and let each VC set that directly?